bazel/repository_locations.bzl

```diff
@@ -1,7 +1,7 @@
 REPOSITORY_LOCATIONS = dict(
-    # envoy 1.24.0, commit: https://github.com/envoyproxy/envoy/commit/15baf56003f33a07e0ab44f82f75a660040db438
+    # envoy 1.24.1-patch1, commit: https://github.com/envoyproxy/envoy/commit/7a6b7cd3fea940b8c2e9d8efe9f21521db842fb8
```

Comment:
the comment is wrong, patch1 is our naming scheme. this should be the revision tag for the upstream envoy commit